src/components/TeamPage/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Header from "../Header";
+import "./TeamPage.css";
+import MemberCard from "../MemberCard";
+import Engineer from "../../assets/images/Engineer.png";
+import Alex from "../../assets/images/Alex.png";
+import Aminah from "../../assets/images/Aminah.png";
+import Greatest from "../../assets/images/Greatest.png";
+import Allan from "../../assets/images/Allan.png";
+import Henry from "../../assets/images/Henry.png";
+import Dorothy from "../../assets/images/Dorothy.png";
+import Steve from "../../assets/images/Steve.png";
+import Paul from "../../assets/images/Paul.png";
+import Colin from "../../assets/images/Colin.png";
+import Mary from "../../assets/images/Mary.png";
+import Rajab from "../../assets/images/Rajab.png";
+import Dora from "../../assets/images/Dora.jpeg";
+import Inno from "../../assets/images/Inno.jpg";
+
+const members = [
+  {id: 1, name:"Engineer Bainomugisha", title:"Team Lead", icon: Engineer},
```

Comment:
`icon` could be a `url/uri` depending on how u have structured it... better named as sth like `imageURL/URI` - uri meaning it's appended to your `base_url` config - so that u don't have the images statically built into the app... we could try a `cdn` like cloudinary or have them stored on our server beforehand... hope that makes sense, too